src/test/java/uk/gov/ons/census/action/schedule/ActionRuleProcessorIT.java

```diff
@@ -119,6 +119,23 @@ public void testIndividualCaseReminderNotSent() throws InterruptedException {
     assertThat(queuedCases.size()).isEqualTo(0);
   }
 
+  @Test
+  public void testCeEstabCase() throws InterruptedException {
+    // Given we have an HI case with a valid Treatment Code.
+    ActionPlan actionPlan = setUpActionPlan();
+    setUpCeEstabCase(actionPlan, 23);
+    setUpCeEstabCase(actionPlan, 77);
```

Comment:
I'd like to enhance this test so that we also have a CE Estab case with `null` as its expected response count